dongamboard/app/src/main/java/com/study/dongamboard/api/APIService.kt

```diff
@@ -1,11 +1,22 @@
 package com.study.dongamboard.api
 
 import com.study.dongamboard.model.PostData
+import com.study.dongamboard.model.request.PostRequest
+import retrofit2.http.Body
 import retrofit2.http.GET
+import retrofit2.http.POST
+import retrofit2.http.Path
 
 interface APIService {
```

Comment:
요거 ApiObject랑 APIService랑 네이밍 통일하면 좋을 것 같아요! 일반적으로 약어도 Api 이런식으로 앞글자만 대문자로 쓰는 경우가 많다고 합니다.
```suggestion
interface ApiService {
```

Reply:
디테일한 네이밍!(늘 창작의 고통을 느끼는 사람) 감사합니다. 코드 정리하면서 반영해 보겠습니다!